server/src/main/java/org/apache/iotdb/db/qp/strategy/LogicalGenerator.java

```diff
@@ -1167,13 +1167,16 @@ private FilterOperator parsePredicate(PredicateContext ctx) {
       return parseOrExpression(ctx.orExpression());
     } else {
       Path path = null;
-      BasicFunctionOperator basic = null;
+      BasicFunctionOperator basic;
       if (ctx.prefixPath() != null) {
         path = parsePrefixPath(ctx.prefixPath());
       }
       if (ctx.suffixPath() != null) {
         path = parseSuffixPath(ctx.suffixPath());
       }
+      if(ctx.TIME() != null || ctx.TIMESTAMP() != null) {
+        path = new Path(SQLConstant.RESERVED_TIME);
```

Comment:
I am not familiar with this part but there is one question, why there is no code about `path = new Path(SQLConstant.RESERVED_TIME);` in current master branch? (Or, why you add such a line of code?).
If no such codes, how current master branch support `select ..... where time > ...`?

Reply:
I guess the current master branch treats "time" as a time series path, it is not distinguished here

Reply:
OK.. As I am not familiar with this part. I do not know whether it is right... You can ignore my question when you decide whether to merge this pr.